src/components/Tooltip.scss

```diff
@@ -2,7 +2,7 @@
 
 // container in body where the actual tooltip is appended to
 .excalidraw-tooltip {
-  position: absolute;
+  position: fixed;
```

Comment:
does this work well with editor offsets?

Reply:
yes this works fine with offsets, you can try out with the example (scroll it and it will render the tooltip correctly)